java/linked-list/app/src/main/java/linked/list/App.java

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
The file name and the class name should be the same. Rename this file to "LinkedList.java"